src/helpers/changelog.js

```diff
@@ -1,17 +1,26 @@
-export const extractChangeLog = obj => {
+export const extractChangeLogMessage = obj => {
   const label = obj.labels.filter(l => {
     return l.name === "changelog";
   });
 
-  return label.length > 0 ? obj.title : null;
+  const getMessage = obj => {
+    const regexMatch = obj.body && obj.body.match(/.*changelog\:(.*)/);
```

Comment:
I think we should require `changelog:` to be start of line and keep it to a single line only to avoid unexpected hits like:

```
Hello this is my changelog: yo!
changelog: My changes
Things not to be in changelog...
```

^^ should produce `My changes`